relay-filter/src/interface.rs

```diff
@@ -108,3 +110,41 @@ impl Filterable for Replay {
         self.user_agent()
     }
 }
+
+impl Filterable for Span {
+    fn csp(&self) -> Option<&Csp> {
+        None
+    }
+
+    fn exceptions(&self) -> Option<&Values<Exception>> {
+        None
+    }
+
+    fn ip_addr(&self) -> Option<&str> {
+        None
+    }
```

Comment:
For events, this is the user IP address. As for `url()`, we should pick a field in span.data and get the value from there (we can sync on this because it's not obvious which field to pick).